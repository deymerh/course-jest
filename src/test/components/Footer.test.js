import React from 'react'
import { mount } from 'enzyme'
import { create } from 'react-test-renderer'
import Footer from '../../components/Footer'

describe('Pruebas en el componente <Footer/>', () => {
  const footer = mount(<Footer />)
  test('render componente Footer', () => {
    expect(footer.length).toEqual(1)
  })
  test('Renderizar el tÃ­tulo', () => {
    expect(footer.find('.Footer-title').text()).toEqual('Platzi Store')
  })  
})

describe('Snapshot del componente <Footer />', () => {
  test('Comprobar UI del compoente Footer', () => {
    const footer = create(<Footer/>)
    expect(footer.toJSON()).toMatchSnapshot()
  })
  
})

