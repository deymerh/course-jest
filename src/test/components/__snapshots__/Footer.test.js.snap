// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot del componente <Footer /> Comprobar UI del compoente Footer 1`] = `
<div
  className="Footer center"
>
  <p
    className="Footer-title"
  >
    Platzi Store
  </p>
  <p
    className="Footer-copy"
  >
    Todos los Izquierdos Reservados
  </p>
</div>
`;
